gnovm/pkg/gnolang/static_analysis_test.go

```diff
@@ -0,0 +1,351 @@
+package gnolang
+
+import "testing"
+
+func TestStaticAnalysisShouldPanic(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("The code did not panic")
+		}
+	}()
+
+	cases := []string{
```

Comment:
These test cases should be organizes as a test table that has a name, so we can execute individual test cases and see which one fails / succeeds